.codecov.yml

```
@@ -37,3 +37,7 @@ ignore:
   - ".github/**/*"                # GitHub workflow files
   - "docs/**/*"                   # Documentation
   - "**/*.md"                     # Markdown files
+  - "go/cmd/gitbak/app_test_helpers.go"  # Test helper functions
+  - "go/internal/git/mocks.go"    # Mock implementations
+  - "go/internal/git/test_helpers.go"  # Test helper functions
+  - "**/*test_helpers.go"         # Any other test helper files
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)